crowdloans/rewards/polkadot.json

```diff
@@ -381,8 +388,8 @@
           "Early Bird: 10% extra rewards if you contribute before 12/23/21.",
           "Referral: 5% referral bonus reward each to referral and referee",
           "Strong Supporter: 10% to users who contribute more than 50,000 DOT",
-          "Quiz Challenge: 2.5% for solving Interlay quiz",
-          "Test Mint Challenge: 2.5% for minting interBTC on testnet",
+          "Quiz Challenge: 2.5% for solving Interlay quiz till 12/23/21",
```

Comment:
Quiz Challenge: 2.5% for solving Interlay quiz before December 23, 2021, 18:00 UTC